js/models/roomcollection.js

```diff
@@ -28,8 +28,15 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: function(model) {
-			return [model.get('active') ? -1 : 0, -(model.get('lastPing'))];
+		comparator: function(modelA, modelB) {
+			var activeA = modelA.get('active'),
+			    activeB = modelB.get('active');
+
+			if (activeA !== activeB) {
+				return activeB - activeA;
+			}
+
+			return modelA.get('lastPing') < modelB.get('lastPing');;
```

Comment:
Remove double `;`

And also do math here? `modelB.get('lastPing') - modelA.get('lastPing')`

Reply:
Done